docs/docs/migration/v0.22.2.md

```diff
@@ -0,0 +1,26 @@
+---
+sidebar_position: 0
+title: v0.22.2
+description: For chains that were scaffolded with Ignite CLI versions lower than v0.22.2, changes are required to use Ignite CLI v0.22.2. 
+---
+
+# Upgrading a blockchain to use Ignite CLI v0.22.2
+
+Ignite CLI v0.22.2 changed the GitHub username from "ignite-hq" to "ignite", which means the imports must be fixed to reflect this change.
+
+1. In your `go.mod` file find the require line for Ignite CLI that starts with `github.com/ignite-hq/cli` and is followed by a version.
+   It looks something like `github.com/ignite-hq/cli v0.22.0`, and replace it by `github.com/ignite/cli v0.22.2`.
+
+2. Open the `app/app.go` file and replace the import line `github.com/ignite-hq/cli/ignite/pkg/openapiconsole` by `github.com/ignite/cli/ignite/pkg/openapiconsole`.
```

Comment:
Instead of giving file by file instructions for .go files, we can suggest them to make a bulk find and search for `github.com/ignite-hq/cli` to be replaced by `github.com/ignite/cli` in the **import statements**.

Which means we can merge 2,3,4 into a single step.

Reply:
Agree, it's probably safer